GeneralsMD/Code/Tools/WorldBuilder/src/brushoptions.cpp

```diff
@@ -87,6 +90,42 @@ void BrushOptions::setHeight(Int height)
 	}
 }
 
+void BrushOptions::setRaiseLowerAmount(Int amount)
+{
+	char buffer[12];
```

Comment:
This buffer is quite small for "%.1f FEET."

Other functions doing that use _MAX_PATH